Sources/Build/BuildDelegate.swift

```diff
@@ -1,7 +1,7 @@
 /*
  This source file is part of the Swift.org open source project
 
- Copyright (c) 2018 Apple Inc. and the Swift project authors
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
```

Comment:
Thanks for taking care to update this!  (a lot of PRs don't)  What we usually do is to add to the range, so this would be `2018-2020` (I think the convention is to use a range rather than list out individual years; also, I think there were actually changes here in 2019).